cli/cephadm/cephadm.py

```diff
@@ -53,3 +53,19 @@ def prepare_host(self, **kw):
         cmd = f"{self.base_cmd} prepare-host {build_cmd_from_args(**kw)}"
 
         return self.execute(sudo=True, cmd=cmd)
+
+    def logs(self, fsid, name, expr=None):
+        """Run the cephadm logs command
+
+        Args:
+            fsid (str): cluster FSID
+            name (str): daemon name
+            expr (str): expression that needs to be filtered
+        """
+        cmd = f"{self.base_cmd} logs --fsid {fsid} --name {name}"
+        if expr:
+            cmd += f" | grep '{expr}'"
```

Comment:
Remove expr, it can be performed after getting the result

Reply:
Done